package.nls.json

```
@@ -63,6 +64,7 @@
     "python.pipenvPath.description": "Path to the pipenv executable to use for activation.",
     "python.poetryPath.description": "Path to the poetry executable.",
     "python.EnableREPLSmartSend.description": "Toggle Smart Send for the Python REPL. Smart Send enables sending the smallest runnable block of code to the REPL on Shift+Enter and moves the cursor accordingly.",
+    "python.sendToNativeREPL.description": "Toggle to send commands to IW Python REPL instead of the terminal.",
```

Are we planning on calling this the IW Python REPL or Native Python REPL or Python REPL? 
```suggestion
    "python.sendToNativeREPL.description": "Toggle to send code to IW Python REPL instead of the terminal on execution. Turning this on will change the behavior for both Smart Send and Run Selection/Line in the Context Menu.",
```

---

Good point, I can change it to Python REPL to be more consistent if that sounds good.